nextcord/interactions.py

```diff
@@ -106,6 +107,11 @@ class Interaction:
         for 15 minutes.
     data: :class:`dict`
         The raw interaction data.
+    client: :class:`Client`
```

Comment:
I don't think adding properties inside the attribute header works, properties docstrings should be enough

Reply:
Yeah, I think you're right about that. I was basing this on okay's PR which had this although their implementation was slightly different (adding it to the data rather than just getting it from state)